lib/Alien/Build/Util.pm

```diff
@@ -65,7 +80,37 @@ sub _mirror
         { unlink "$dst" }
         my $target = readlink "$src";
         Alien::Build->log("ln -s $target $dst") if $opt->{verbose};
-        symlink($target, $dst) || die "unable to symlink $target => $dst";
```

Comment:
I would like to keep this simplified behavior when $^O is not `msys` or `cygwin`.